examples/keras_simple.py

```diff
@@ -0,0 +1,81 @@
+"""
+Optuna example that optimizes a neural network classifier configuration for the
+MNIST dataset using Keras.
+
+In this example, we optimize the validation accuracy of MNIST classification using
+Keras. We optimize the filter and kernel size, kernel stride and layer activation.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python keras_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize keras_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import numpy as np
+
+from sklearn.metrics import accuracy_score
+
+from keras.datasets import mnist
+from keras.layers import Conv2D
+from keras.layers import Dense
+from keras.layers import Flatten
+from keras.models import Sequential
+from keras.optimizers import Adam
+
+import optuna
+
+
+def objective(trial):
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
```

Comment:
Please truncate the datasets instead of full datasets as can be seen in [`examples/pruning/keras_integration.py`](https://github.com/pfnet/optuna/blob/master/examples/pruning/keras_integration.py#L23-L24).